week4/weather-forecast/src/components/PageLayout.jsx

```diff
@@ -0,0 +1,29 @@
+import styled from "styled-components";
+import React from "react";
+import Header from "./Header";
+import SearchForm from "./SearchForm";
+
+const PageLayout = (props) => {
+	const { children } = props;
```

Comment:
```suggestion
const PageLayout = ({ children }) => {
```

props가 하나라면 그냥 이렇게 적어줘도 깔끔하게 괜찮지 않을까요?